pkgs/development/libraries/libbluray/default.nix

```diff
@@ -20,7 +20,10 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-wksPQcW3N7u2XFRP5jSVY3p3HBClGd/IAudp8RK0O3U=";
   };
 
-  patches = optional withJava ./BDJ-JARFILE-path.patch;
+  patches = [
+    (optional withJava ./BDJ-JARFILE-path.patch)
+    (optional withJava ./libbluray-1.3.1-Fix-build-failure-after-Oracle-Java-CPU-for-April-2022.patch)
+  ];
 
   nativeBuildInputs = [ pkg-config autoreconfHook ]
                       ++ optionals withJava [ ant ]
```

Comment:
```suggestion
    ++ optionals withJava [ ant ]
```

Reply:
This isn't code I committed. Shouldn't this be done in a separate commit/PR instead?

Reply:
This looks worse, IMO.

A better solution would be

```nix
nativeBuildInputs = [
  pkg-config
  autoreconfHook
]
++ optionals withJava [ ant ]
```

But either way, this isn't code this change touched, so do we really have to reformat it? Plenty of files around Nixpkgs use bespoke formatting, and I don't touch it if I don't have to, only focusing on what I change.

Reply:
> This isn't code I committed. Shouldn't this be done in a separate commit/PR instead?

Just do it here, its fine. Especially another PR for that is overkill.

---

> But either way, this isn't code this change touched, so do we really have to reformat it? Plenty of files around Nixpkgs use bespoke formatting, and I don't touch it if I don't have to, only focusing on what I change.

No, but it is nice to gradually improve things and not keep old outdated things.

Reply:
> No, but it is nice to gradually improve things and not keep old outdated things.

I'm not sure why you focused on just that attribute, then -- there are multiple with the same formatting. There is no point in changing one of them and not the rest, and it's not even code that's touched by this PR. Reformatting in a separate commit would be nice, but it's not required.